app/controllers/hq/sponsorships_controller.rb

```diff
@@ -1,5 +1,19 @@
 class Hq::SponsorshipsController < HqController
 
+  def create
+    params.permit :orphan_id
```

Comment:
This line should be removed. We are not doing mass assignment so there is no need for this.